system_metrics_collector/src/system_metrics_collector/default_config.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This file duplicates constants.hpp, with some minor namespace changes.

Reply:
Removed `default_config.hpp`.